api/services/scosService.ts

```diff
@@ -10,6 +10,7 @@ async function makeSCOSRequest(body: any): Promise<any> {
     body,
     headers: {
       'Content-Type': 'text/plain',
+      'User-Agent': 'needed something for scos',
```

Comment:
Lol...you could call this something more official like `SCOS Api`

Reply:
changed